cocoa/tests_backend/widgets/table.py

```diff
@@ -95,7 +95,7 @@ def row_position(self, row):
         )
 
     async def select_all(self):
-        await self.type_character("A", alt=True),
+        (await self.type_character("A", alt=True),)
```

Comment:
This seems like a typo in the original - the comma is redundant.
```suggestion
        await self.type_character("A", alt=True)
```